help/desktop-app-install-guide.md

```diff
@@ -98,6 +98,22 @@ automatically to future versions.
 
 3. Run Zulip from your app launcher, or with `zulip` from a terminal.
 
+#### Flathub
+
+1. Make sure you have [Flatpak](https://flathub.org/setup) installed on your system.
+
+2. Use the following command to install Zulip from Flathub:
```

Comment:
Maybe tweak this step and drop the last line "You can always check the official...."

```suggestion
2. Use the following command from the official
[Flathub page](https://flathub.org/apps/org.zulip.Zulip) to install Zulip:
```